src/index.js

```diff
@@ -2,10 +2,19 @@ import SauceTunnel from 'sauce-tunnel';
 import wd from 'wd';
 import promisify from 'es6-promisify';
 import { Promise } from 'es6-promise';
+import request from 'request';
+import denodeify from 'denodeify';
+import wait from './utils/wait';
+import util from 'util';

```

Comment:
move import utils up. Our modules (like `./utils/wait`) should be placed after external modules imports.
And maybe it's better to use `import { format } from 'util'`